### ISSUE TITLE
Determine failure on lab

### ISSUE THREAD

**Issue by carter2422** [CGCookie/cgc-rcp-stripe-checkout#2]
Stripe Checkout failed on first signup attempt on Lab. White screened. 

The customer was created in Stripe, but no subscription or charges were created. This means it also didn't update in RCP, so it likely bailed right after sending the initial customer data.

**Comment by carter2422** [comment]
It failed because there was no such plan in Stripe as _Citizen Quarterly_ in Stripe:

```
Sep 18 10:49:24 cgcookie-staging-remote-syslog2-1 hhvm.error.log:  \nFatal error: Uncaught exception 'Stripe\\Error\\InvalidRequest' with message 'No such plan: citizenquarterly' in /home/alpha/deployments/cgcookielab.com/releases/20150918153507/web/app/plugins/cgc-rcp-stripe-checkout/vendor/stripe/stripe-php/lib/ApiRequestor.php:102 from API request 'req_70S1k3JSsDSbOM'\nStack trace:\n#0 /home/alpha/deployments/cgcookielab.com/releases/20150918153507/web/app/plugins/cgc-rcp-stripe-checkout/vendor/stripe/stripe-php/lib/ApiRequestor.php(216): Stripe\\ApiRequestor->handleApiError()\n#1 /home/alpha/deployments/cgcookielab.com/releases/20150918153507/web/app/plugins/cgc-rcp-stripe-checkout/vendor/stripe/stripe-php/lib/ApiRequestor.php(60): Stripe\\ApiRequestor->_interpretResponse()\n#2 /home/alpha/deployments/cgcookielab.com/releases/20150918153507/web/app/plugins/cgc-rcp-stripe-checkout/vendor/stripe/stripe-php/lib/ApiResource.php(105): Stripe\\ApiRequestor->request()\n#3 /home/alpha/deploy
```

**Comment by carter2422** [comment]
Will fix via #3
